src/it/scala/com/ing/wbaa/gargoyle/proxy/provider/AuthorizationProviderRangerItTest.scala

```diff
@@ -13,6 +13,7 @@ class AuthorizationProviderRangerItTest extends AsyncWordSpec with DiagrammedAss
   val s3Request = S3Request(
     AwsRequestCredential(AwsAccessKey("accesskey"), Some(AwsSessionToken("sessiontoken"))),
     Some("demobucket"),
+    None,
     Read
   )
 
```

Comment:
Currently all tests in this file are for case where objectRoot is None. Would be nice to write some tests for the list-bucket, create-bucket as well, so we cover all case statements in AuthorizationProviderRanger.

To overwrite the config settings for allow-list-buckets you can state on row 33 sth like:
`override def rangerSettings: GargoyleRangerSettings = GargoyleRangerSettings(testSystem) {
    override val ASetting = "whatever"
}`